midend/lib/Utils/DIPUtils.cpp

```diff
@@ -410,6 +411,185 @@ Value customTanVal(OpBuilder &builder, Location loc, Value angleVal) {
   return builder.create<arith::DivFOp>(loc, sinVal, cosVal);
 }
 
+SmallVector<Value, 6> getRotationMatrix(OpBuilder& builder, Location loc, Value centerX, Value centerY, Value angle, Value scale) {
+  Value alpha0 = builder.create<math::CosOp>(loc, angle);
+  Value alpha = builder.create<arith::MulFOp>(loc, alpha0, scale);
+  Value beta0 = builder.create<math::SinOp>(loc, angle);
+  Value beta = builder.create<arith::MulFOp>(loc, beta0, scale);
+  Value oneMinusAlpha = builder.create<arith::SubFOp>(loc, builder.create<arith::ConstantOp>(loc, builder.getF32FloatAttr((float)1.)), alpha);
+  SmallVector<Value, 6> mat;
```

Comment:
This one as well variable declared on lines 425, 426 and 429 seem redundant, are they not?